### PR TITLE
build(ci): enforce coverage check

### PR DESCRIPTION
**Pull request by qweeah**
**What this PR does / why we need it**:
This enforces the test coverage check by 75% lines per project

### CODE REVIEW

File: .github/codecov.yml [hunk 1]
```diff
@@ -15,4 +15,5 @@ coverage:
   status:
     project:
       default:
-        target: 60%
\ No newline at end of file
+        target: 75%
+        if_ci_failed: error
```
Comment (TerryHowe) [inline .github/codecov.yml:19]: Maybe not such a big jump in coverage right away?

Comment (qweeah) [reply .github/codecov.yml:19]: Seems a big change but actually we already achieved ~80% test coverage. Considering we already have 2900+(3600 line * 80%) lines covered, to make coverage lower than 75% means 250 lines of new code were added without testing.

**Review by TerryHowe — APPROVED** [review]
/lgtm

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Pull request merged**